### PR TITLE
RobloxCorporation.RobloxStudio version 1, 6, 0, 417678

### PR DESCRIPTION
**Pull request by markandpreston1**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40073)

### CODE REVIEW

File: manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: RobloxCorporation.RobloxStudio
+PackageVersion: 1, 6, 0, 417678
```
Comment (ItzLevvie) [inline manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.installer.yaml:4]:
```suggestion
PackageVersion: 1.6.0.426194
```

File: manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: RobloxCorporation.RobloxStudio
+PackageVersion: 1, 6, 0, 417678
```
Comment (ItzLevvie) [inline manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.6.0.426194
```

File: manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: RobloxCorporation.RobloxStudio
+PackageVersion: 1, 6, 0, 417678
```
Comment (ItzLevvie) [inline manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.yaml:4]:
```suggestion
PackageVersion: 1.6.0.426194
```

File: manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: RobloxCorporation.RobloxStudio
+PackageVersion: 1, 6, 0, 417678
+PackageLocale: en-US
+Publisher: Markandpreston
```
Comment (ItzLevvie) [inline manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.locale.en-US.yaml:6]:
```suggestion
Publisher: Roblox Corporation
```

File: manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: RobloxCorporation.RobloxStudio
+PackageVersion: 1, 6, 0, 417678
+PackageLocale: en-US
+Publisher: Markandpreston
+PackageName: Roblox Studio
+License: Copyright © 2020 Roblox Corporation. All rights reserved.
+ShortDescription: Go to https://en.wikipedia.org/wiki/Roblox, Page 1,1for about Roblox Studio. And no, i didin't make this. its made by Roblox Corporation, i am just publishing it to winget. anyways bye. I will keep updating this to match the latest version.
```
Comment (ItzLevvie) [inline manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.locale.en-US.yaml:9]:
```suggestion
ShortDescription: Roblox Studio lets you create anything and release with one click to smartphones, tablets, desktops, consoles, and virtual reality devices.
```

File: manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: RobloxCorporation.RobloxStudio
+PackageVersion: 1, 6, 0, 417678
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://setup.rbxcdn.com/RobloxStudioLauncherBeta.exe
+  InstallerSha256: FBFC6FF875BF556AE2F53D585DAA0E98349D8106E7EAAA781EE10143229A46D0
```
Comment (ItzLevvie) [inline manifests/r/RobloxCorporation/RobloxStudio/1, 6, 0, 417678/RobloxCorporation.RobloxStudio.installer.yaml:9]:
```suggestion
  InstallerSha256: FF62991C65F22AC14AA9C53AB70D2D720A3C8A628E71CB305F46478310C513FD
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
I believe this package will always result in a hash mismatch because the CDN appears to be a dynamic CDN. This means that users  will get different versions and different file hashes depending on exactly when and where they are downloading it from.

I've had the exact same issue in the past - https://github.com/microsoft/winget-pkgs/pull/35548/ and don't believe this package can be added to winget without a static url to a specific version

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/40073#issuecomment-1006943741

**Pull request closed without merging**